tests/test_persona_service.py

```
@@ -0,0 +1,82 @@
+    with open(file_path, "w", encoding="utf-8") as f:
+        json.dump(persona_data, f)
+    # set env variable to temp file path
+    monkeypatch.setenv("PERSONA_FILE", str(file_path))
+    service = PersonaService()
+    persona = service.load_persona()
+    assert isinstance(persona, Persona)
+    assert persona.role == "Test Role"
+
+
+# test get_persona returns loaded persona
+def test_get_persona(monkeypatch):
```

Ruff (0.15.0)

[warning] Unused function argument: `monkeypatch`

(ARG001)